{"classFile":"CloneFailedExceptionTest","className":"CloneFailedExceptionTest","packages":"org.apache.commons.lang3.exception","methodInfos":[{"md5":"K8qhja5rutvnGPVLvJT6Dg==","methodName":"testWithCauseAndMessage","parameters":"[]"},{"md5":"TI8isUEajxhIAL/8H8x2ug==","methodName":"testWithoutCause","parameters":"[]"},{"md5":"m24VRynyZqzqfXsmmoB/zg==","methodName":"testWithoutMessage","parameters":"[]"}],"addLines":[53,64,74],"delLines":[53,64,74],"type":"MODIFY","path":"src/test/java/org/apache/commons/lang3/exception/CloneFailedExceptionTest.java","pro_Commit":"7b6ab58a018b5f6608ea07bc414d3f34a576c6a2","pro_Date":"2010-02-11 07:22:05","sha1":"3ea05e7ea3980525fedf8029c2b54f66b3238c9f","file_Date":"2018-10-14 03:09:37","projectName":"commons-lang","commitID":"b3d86ca08a22afb47338dea0a3fcad2a58ece49c"}